components/layouts/PageFooter/index.vue

```diff
@@ -16,31 +35,114 @@
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
 import FooterSection from './FooterSection.vue'
+import FooterIconsSection from './FooterIconsSection.vue'
+import AppLink from '~/components/ui/AppLink.vue'
 
-import { FOOTER_ELEMENTS } from '~/constants/menuLinks'
+import {
+  FOOTER_ELEMENTS,
+  SECONDARY_FOOTER_LINKS,
+  STAY_CONNECTED_LINKS
+} from '~/constants/menuLinks'
 
 @Component({
   components: {
-    FooterSection
+    FooterSection,
+    FooterIconsSection,
+    AppLink
   }
 })
 export default class extends Vue {
   footerElements = FOOTER_ELEMENTS
+  stayConnectedElements = STAY_CONNECTED_LINKS
+  secondaryFooterLinks = SECONDARY_FOOTER_LINKS
 }
 </script>
 
 <style lang="scss" scoped>
-.page-footer {
+@import '~carbon-components/scss/globals/scss/typography';
+
+.main-footer, .secondary-footer {
+  /*
+  * The footer should be framed() (max-width of 1100px = 68.75rem)
+  * If we framed it, the backgroud fit the framed size, not all
+  * That's why we calculate the padding
+  */
+  padding: $spacing-07 calc((100vw - 68.75rem) / 2);
   display: flex;
-  flex-direction: row;
-  color: $text-02;
+  justify-content: space-between;
+}
 
-  &--framed {
-    @include framed();
-  }
+.main-footer {
+  background-color: $cool-gray-10;
+  overflow: hidden;
 
   @include mq($until: medium) {
-    display: block;
+    flex-direction: column;
+  }
+
+  &__logo {
+    height: 32px;
+    width: 112px;
+    margin: 0 $spacing-07;
+
+    @include mq($until: medium) {
+      margin-bottom: $spacing-07;
+    }
+  }
+
+  &__sections {
+    display: grid;
+    grid-template-columns: repeat(6, 0.5fr);
+    margin: 0 $spacing-07;
+    row-gap: $spacing-07;
+
+    @include mq($until: medium) {
+      grid-template-columns: repeat(3, 0.5fr);
+      grid-template-rows: repeat(2, 0.5fr);
+    }
+
+    @include mq($from:medium, $until: large) {
+      grid-template-columns: repeat(5, 0.5fr);
+      grid-template-rows: repeat(2, 0.5fr);
+
+      :nth-child(6) {
```

Comment:
I'd say it is preferable to use `last-child` here. There would be more scalable solutions but I think using `last-child` improves the intentionality.